src/modules/views/Public/PublicFolderView.spec.jsx

```diff
@@ -67,13 +67,25 @@ jest.mock('components/pushClient', () => ({
 useSharingContext.mockReturnValue({ byDocId: [] })
 
 describe('Public View', () => {
+  beforeAll(() => {
+    // Mock console methods to suppress redux-logger output
```

Comment:
```suggestion
```

also it would be nice to have new tests for the useFolderSort hook and the new props we add